django_cas_ng/views.py

```diff
@@ -109,8 +109,9 @@ def login(request, next_page=None, required=False):
         elif settings.CAS_RETRY_LOGIN or required:
             return HttpResponseRedirect(_login_url(service))
         else:
-            error = "<h1>Forbidden</h1><p>Login failed.</p>"
-            return HttpResponseForbidden(error)
+            #error = "<h1>Forbidden</h1><p>Login failed.</p>"
+            #return HttpResponseForbidden(error)
+            return HttpResponseRedirect(next_page)
```

Comment:
Is this part of the spec?  If so, can you leave a link in a comment to the place in the specification where this is stated?

If not, can you explain why this was changed?